README.md

```diff
@@ -0,0 +1,6 @@
+Toner
+------------
+
+![Toner screenshot](Toner.png "Toner")
+
+Toner is a tool to generates tones for aligning levels in your playback system. 

```

Comment:
It's "generate". Also, since the heading is "Toner", it's OK to shorten to "A tool to generate..." I think.

Reply:
Sorry... my keyboard it's a little bit crazy today. Fixed it...